apisix/plugins/authz-keycloak.lua

```diff
@@ -722,12 +722,12 @@ local function generate_token_using_password_grant(conf,ctx)
     if not username then
         local err = "username is missing."
         log.error(err)
-        return 422, err
+        return 422, {message = err}
```

Comment:
Would be better to add test to cover this branch?

Reply:
add